### PR TITLE
Pin 5270: Tenant event consumer

### PR DESCRIPTION
**Pull request by eduwardo**
## Description

Implemented `tenant-event-consumer` microservice to handle a consumer to populate tenants table

## Changes Made
- Added tenant-event-consumer package.
- Updated operations package.
- Updated commons package.

### CODE REVIEW

File: packages/tenant-event-consumer/src/handlers/messageHandlerV2.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        if (!tenant) {
+          throw kafkaMessageMissingData(config.kafkaTopic, event.type);
+        }
+
+        await operationsService.saveTenant(
+          { ...correlationIdToHeader(ctx.correlationId) },
+          {
+            tenantId: tenant.id,
+            externalId: tenant.externalId?.value || "TODO",
+            origin: tenant.externalId?.origin || "TODO",
```
Comment (eduwardo) [inline packages/tenant-event-consumer/src/handlers/messageHandlerV2.ts:33]: Make sure that are not mandatory so sql can be updated to allow null

File: packages/tenant-event-consumer/src/handlers/messageHandlerV1.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+        const { tenant } = evt.data;
+
+        await operationsService.saveTenant(
+          { ...correlationIdToHeader(ctx.correlationId) },
+          {
+            tenantId: tenant.id,
+            externalId: tenant.externalId?.value || "TODO",
+            origin: tenant.externalId?.origin || "TODO",
+            name: tenant.name || "TODO",
```
Comment (eduwardo) [inline packages/tenant-event-consumer/src/handlers/messageHandlerV1.ts:35]: Make sure that are not mandatory so sql can be updated to allow null

Comment (lorenzo-pa) [reply packages/tenant-event-consumer/src/handlers/messageHandlerV1.ts:35]: Ok, let's mark the columns to "nullable"

File: packages/tenant-event-consumer/src/handlers/messageHandlerV2.ts (hunk 1 above)
Comment (lorenzo-pa) [reply packages/tenant-event-consumer/src/handlers/messageHandlerV2.ts:33]: Ok, let's mark the columns to "nullable"

**Review by lorenzo-pa — APPROVED** [review]

**Pull request merged**